src/main/scala/wdl4s/Task.scala

```diff
@@ -30,6 +30,38 @@ object Task {
       case x: Ast => ParameterCommandPart(x, wdlSyntaxErrorFormatter)
     }
 
+    commandTemplate collect { case x: ParameterCommandPart => x } foreach { p =>
+      p.expression.ast.findTerminals("identifier") foreach {variable =>
```

Comment:
this `findTerminals("identifier")` is too permissive.  It also catches function names... I need to fix that.